source/process/community-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+
+Increase Mattermost usage through more numerous and larger deployments via vibrant open source community by contributing features that would not otherwise be offered
+
+Principles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Contributors get a worthwhile benefit from their contributions from the feature directly, and/or from social recognition
+- Help Wanted issues should have a low barrier to contribute
+
+Creating help wanted issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before creating a help wanted issue in the `platform repository <https://github.com/mattermost/platform>`_, confirm the issue you're planning to add has a JIRA ticket associated with it
+
+- JIRA can be used to prioritize help wanted issues internally
+- each issue goes through `Triage meeting <https://docs.mattermost.com/process/training.html#triage-meeting>`_ to ensure the ticket is clear and is approved by at least one developer and product manager
+
+When creating a help wanted issue, please ensure it contains the following (`see a sample help wanted issue here <https://github.com/mattermost/platform/issues/4755>`_):
+
+- is well-defined and unambiguous
+ 
+ - create the issue with the mindset that the contributor might have no or limited experience with the Mattermost code base and limited exposure to the Mattermost product
+    
+- has an introductory section, with
+ 
+ - an invitation to join the `Contributors channel <https://pre-release.mattermost.com/core/channels/tickets>`_
+ - an invitation to join the `Developers channel <https://pre-release.mattermost.com/core/channels/developers>`_ for technical questions
+ - a link to the `developer’s guide <https://docs.mattermost.com/guides/developer.html>`_ for `machine setup <https://docs.mattermost.com/developer/developer-setup.html>`_ and `developer workflow <https://docs.mattermost.com/developer/developer-flow.html>`_
+ - a link to the `code contribution guidelines <https://docs.mattermost.com/developer/contribution-guide.html>`_
+    
+- has a well-defined difficulty level, both in the description and with the corresponding label
+ 
+ - Introductory: for someone new to the codebase. Guideline: 2-mana tickets are typically introductory
+ - Intermediate: for someone with a general knowledge of the codebase. Guideline: 4-mana tickets are typically introductory
+ - Advanced: for someone with a strong grasp of the codebase, e.g. implementing a major feature. Guideline: 8-mana tickets (or higher) are typically introductory     
+    
+- has a “Go” or “ReactJS” label
+ 
+ - helps contributors know what programming languages will be involved
+ - motivates contributors to work on specific tickets, e.g. someone learning Go might be motivated to tackle introductory Go tickets
+ - in general, if the work is all UI (client side), add a ReactJS label - if it's server side then add a Go label.
+
+- has a link to the corresponding JIRA ticket 
+
+- gets noted in the corresponding JIRA ticket
```

Comment:
maybe change to gets added to the Jira ticket description, because I think devs were sometimes missing it in the comments section